tests/test_e2e_18_mef_eline.py

```diff
@@ -0,0 +1,219 @@
+import requests
+import time
+
+
+from .helpers import NetworkTest
+
+CONTROLLER = '127.0.0.1'
+KYTOS_API = 'http://%s:8181/api' % CONTROLLER
+
+class TestE2EMefEline:
+    net = None
+
+    @classmethod
+    def setup_class(cls):
+        cls.net = NetworkTest(CONTROLLER, topo_name="ring")
+        cls.net.start()
+        cls.net.restart_kytos_clean()
+        cls.net.wait_switches_connect()
+        time.sleep(5)
+
+    @classmethod
+    def teardown_class(cls):
+        cls.net.stop()
+
+    def setup_method(self, method):
+        self.net.start_controller(clean_config=True, enable_all=True)
+        self.net.wait_switches_connect()
+        time.sleep(10)
+
+    def teardown_method(self, method):
+        for link in self.net.net.links:
+            self.net.net.configLinkStatus(
+                link.intf1.node.name,
+                link.intf2.node.name,
+                "up"
+            )
+
+    def is_tag_used_by_interface(self, interface_id, tag_type, value):
+        api_url = KYTOS_API + "/kytos/topology/v3/interfaces/tag_ranges"
+        response = requests.get(api_url)
+        assert response.status_code == 200, response.text
+
+        data = response.json()
+
+        interface_tags = data[interface_id]
+        available_tags = interface_tags["available_tags"][tag_type]
+        # tag_ranges = interface_tags["tag_ranges"][tag_type]
+
+        for range_start, range_end in available_tags:
```

Comment:
Linear search is acceptable here since it's only used by this test suite and the underlying ranges list is expected to be small.